### ISSUE TITLE
Is it normal the training loss looks like this?

### ISSUE THREAD

**Issue by xfffrank** [swc-17/SparseDriveV2#11]
<img width="1515" height="444" alt="Image" src="https://github.com/user-attachments/assets/caf64dea-664a-4598-a3bc-2616efe8fce8" />

It converged quickly and then kept oscillating

**Comment by swc-17** [comment]
You can smooth the loss curve or directly check the metric.

**Comment by xfffrank** [comment]
**The metric barely improved after a few epochs.** I think it somehow related to the oscillating loss. Did you observe the same pattern in your experiments? @swc-17 

In my case, the backbone and how the scorer interacts with the scene features are different. So I'm not sure if the oscillating loss is expected in Sparse Drive V2

**Comment by xfffrank** [comment]
<img width="623" height="467" alt="Image" src="https://github.com/user-attachments/assets/d56c282f-1f29-4ee0-85ca-5fcb67ecc9fd" />

In my case, one of the metrics is minADE. And it looks like this.

**Comment by swc-17** [comment]
<img width="1049" height="457" alt="Image" src="https://github.com/user-attachments/assets/59a5feb3-c631-4645-9ed4-854a52788ee0" /> In my case,  10 epoch training on NAVSIM is enough to converge.

**Comment by xfffrank** [comment]
Thanks!
